extra-utils/hdsentinel/autobuild/build

```diff
@@ -0,0 +1,21 @@
+abinfo Detect architecture
+hdsentinelVer="017"
+hdsentinelBaseName="hdsentinel-$hdsentinelVer"
+if [[ "${CROSS:-$ARCH}" = "amd64" ]]; then
+    hdsentinelFileName="$hdsentinelBaseName-x64"
+elif [[ "${CROSS:-$ARCH}" = "i386" ]]; then
+    hdsentinelFileName="$hdsentinelBaseName"
+elif [[ "${CROSS:-$ARCH}" = "armel" ]]; then
+    hdsentinelFileName="$hdsentinelBaseName-arm"
+else
+    exit 1
+fi
+
+abinfo Downloading pre-built binaries
+wget https://www.hdsentinel.com/hdslin/"$hdsentinelFileName".gz
+gzip -d "$hdsentinelFileName".gz
+chmod 0755 "$hdsentinelFileName"
+
+abinfo Deploy files
+mkdir -p "$PKGDIR"/usr/bin
+mv "$hdsentinelFileName" "$PKGDIR"/usr/bin/hdsentinel
```

Comment:
This will not be visible to users on a desktop environment if it is a GUI program. You'll need to add a `.desktop` file in `/usr/share/applications/` to present it in the application menu.

Reply:
Thanks for that, but it's CLI only...